ksm-throttler/update.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+# -*- mode: shell-script; indent-tabs-mode: nil; sh-basic-offset: 4; -*-
+# ex: ts=8 sw=4 sts=4 et filetype=sh
+#
+# Automation script to create specs to build ksm-throttler.
+# Default: Build is the one specified in file configure.ac
+# located at the root of the repository.
+set -e
+
+source ../versions.txt
+source ../scripts/pkglib.sh
+
+SCRIPT_NAME=$0
+SCRIPT_DIR=$(dirname $0)
+PKG_NAME="kata-ksm-throttler"
+VERSION=$ksm_throttler_version
+HASH=$ksm_throttler_hash
+RELEASE=$(cat release)
+
+GENERATED_FILES=(_service kata-ksm-throttler.spec kata-ksm-throttler.dsc debian.control debian.rules)
+STATIC_FILES=(debian.compat)
+
+COMMIT=false
+BRANCH=false
+LOCAL_BUILD=false
+OBS_PUSH=false
+VERBOSE=false
+
+# Parse arguments
+cli "$@"
+
+[ "$VERBOSE" == "true" ] && set -x || true
+PROJECT_REPO=${PROJECT_REPO:-home:katacontainers:release/ksm-throttler}
```

Comment:
move to one variable:
home:
katacontainers -> obs_project?
release -> obs_subproject

Reply:
Add move the definition to pkglib to avoid repetition